Game2048/Game/Models/Abstracts/ActionClass.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace AbstractActionClass
```

Comment:
It is possible that this name could be confused with a class name since it contains "Abstract" and "Class" words. Namespaces purpose is to group classes/enums/interfaces, thus, a group name like `Actions` could be a better suit